property_tests/tests/stable_b_tree_map/test/get.ts

```diff
@@ -0,0 +1,78 @@
+import fc from 'fast-check';
+import { deepEqual } from 'fast-equals';
+
+import { StableBTreeMapArb } from '../../../arbitraries/stable_b_tree_map_arb';
+import { getActor } from '../../../../property_tests';
+import { Test } from '../../../../test';
+import { UniqueIdentifierArb } from '../../../arbitraries/unique_identifier_arb';
+import { QueryMethod } from '../../../arbitraries/canister_methods/query_method_arb';
+
+export const GetTestArb = fc
+    .tuple(UniqueIdentifierArb('stableBTreeMap'), StableBTreeMapArb)
+    .map(([functionName, stableBTreeMap]): QueryMethod => {
+        const imports = new Set([
+            ...stableBTreeMap.param0.src.imports,
+            ...stableBTreeMap.param1.src.imports,
+            'Opt',
+            'stableJson',
+            'StableBTreeMap'
+        ]);
+
+        const paramCandidTypeObjects = [
+            stableBTreeMap.param0.src.candidTypeAnnotation,
+            stableBTreeMap.param1.src.candidTypeAnnotation
+        ].join(', ');
+
+        const returnCandidType = `Opt(${stableBTreeMap.param1.src.candidTypeAnnotation})`;
+        const body = generateBody(stableBTreeMap.name, stableBTreeMap.body);
+
+        const test = generateTest(
+            functionName,
+            stableBTreeMap.param0.agentArgumentValue,
+            stableBTreeMap.param1.agentArgumentValue
+        );
+
+        return {
+            imports,
+            globalDeclarations: [
+                ...stableBTreeMap.param0.src.variableAliasDeclarations,
+                ...stableBTreeMap.param1.src.variableAliasDeclarations
+            ],
+            sourceCode: `${functionName}: query([${paramCandidTypeObjects}], ${returnCandidType}, (param0, param1) => {
+                ${body}
+            })`,
+            tests: [test]
+        };
+    });
+
+function generateBody(
+    stableBTreeMapName: string,
+    stableBTreeMapBody: string
+): string {
+    return `
+        ${stableBTreeMapBody}
```

Comment:
Calling this `stableBTreeMapBody` doesn't seem quite right. It's creating the map, so I'd lean more to something more like constructor. It feels like it's going to be like the following two lines, but then isn't. I'm not explaining well though so let's talk about this IRL if you'd like.